apps/src/levelbuilder/ai-iteration-tools/StudentCodeDatasetMaker.tsx

```diff
@@ -8,14 +9,20 @@ import {fetchStudentCodeSamples} from './StudentCodeSamplesApi';
 interface StudentCodeSample {
   projectId: string;
   studentCode: string | undefined;
+  codeVersion?: string;
   userId: number | undefined;
+  aiEvaluation?: string;
+  aiReasoning?: string;
+  evaluationCriteria?: string;
 }
 
 const StudentCodeDatasetMaker: React.FC = () => {
   const [datasetName, setDatasetName] = useState<string>('');
   const [levelId, setLevelId] = useState<string>('');
-  const [scriptId, setScriptId] = useState<string>('');
+  const [unitId, setUnitId] = useState<string>('');
```

Comment:
I've been swapping "unit" for "script" as it comes up since based on this [comment](https://github.com/code-dot-org/code-dot-org/pull/64321#discussion_r1984177089)